Makefile

```diff
@@ -127,7 +127,10 @@ audit:
 test: test-unit test-e2e test-wasm
 
 test-e2e:
-	RUST_BACKTRACE=1 $(cargo) test e2e -- --test-threads=1
+	RUST_BACKTRACE=1 $(cargo) test e2e \
```

Comment:
is this change needed? It seems to work with the defaults

Reply:
I did that at first to use the mock eth node, but I forgot the we compile as part of the e2e test. So it is not needed.